src/createXcmTypes/util/getAssetHubAssetId.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright 2023 Parity Technologies (UK) Ltd.
+
+import { ApiPromise } from '@polkadot/api';
+
+import { ASSET_HUB_CHAIN_ID } from '../../consts';
+import { BaseError } from '../../errors';
+import { Registry } from '../../registry';
+import { foreignAssetMultiLocationIsInRegistry } from './foreignAssetMultiLocationIsInRegistry';
+import { getChainIdBySpecName } from './getChainIdBySpecName';
+
+/**
+ * Returns the correct asset id for a valid AssetHub token symbol integer id
```

Comment:
This comment is a bit confusing. I think the punctuation just needs to be fixed and it will be perfect.